api/CacheStorage.json

```diff
@@ -0,0 +1,340 @@
+{
+  "api": {
+    "CacheStorage": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CacheStorage",
+        "support": {
+          "chrome": {
+            "version_added": "40",
+            "notes": [
+              "Accessible from <a href='https://developer.mozilla.org/docs/Web/API/Window' title='The window object represents a window containing a DOM document; the document property points to the DOM document loaded in that window.'><code>Window</code></a> from version 43",
```

Comment:
Better: `"Accessible from <code>Window</code> from version 43."` (without the link).